test/k8sT/Policies.go

```diff
@@ -401,17 +401,29 @@ var _ = Describe("K8sPolicyTest", func() {
 			Expect(err).Should(BeNil())
 		})
 
+		getPolicyCmd := func(policy string) string {
+			return fmt.Sprintf("%s=%s %s=%s",
+				helpers.KubectlPolicyNameLabel, policy,
+				helpers.KubectlPolicyNameSpaceLabel, helpers.DefaultNamespace)
+		}
+
 		AfterEach(func() {
-			kubectl.Delete(kubectl.ManifestGet(deployment)).ExpectSuccess(
-				"Guestbook deployment cannot be deleted")
+
 			kubectl.Delete(kubectl.ManifestGet(webPolicy)).ExpectSuccess(
 				"Web policy cannot be deleted")
 			kubectl.Delete(kubectl.ManifestGet(redisPolicyDeprecated)).ExpectSuccess(
 				"Redis deprecated policy cannot be deleted")
+			kubectl.Delete(kubectl.ManifestGet(deployment)).ExpectSuccess(
+				"Guestbook deployment cannot be deleted")
```

Comment:
This is halfway there, but still not quite there yet. I will clarify more what I think is needed here. Calls to `kubectl delete...` are asynchronous. The resources are not truly "deleted" after the call to `kubectl delete` returns. We need to wait until the pods are no longer present in the output of `kubectl get pods`, to ensure that the pods are fully cleaned up while this test is still running. I'm cautious about leftover cruft from prior tests interfering with test runs.